ui/components.go

```diff
@@ -630,8 +630,78 @@ func (page *pageCommon) initSelectAccountWidget(wallAcct map[int][]walletAccount
 	}
 }
 
+func ticketStatusTooltip(gtx C, c *pageCommon, rect image.Rectangle, t *wallet.Ticket, tooltip *decredmaterial.Tooltip) layout.Dimensions {
+	inset := layout.Inset{
+		Top:   values.MarginPadding15,
+		Right: unit.Dp(-150),
+		Left:  values.MarginPadding15,
+	}
+	return tooltip.Layout(gtx, rect, inset, func(gtx C) D {
+		st := ticketStatusIcon(c, t.Info.Status)
+		var title, message, message2 string
+		switch t.Info.Status {
+		case "UNMINED":
+			title = "This ticket is waiting in mempool to be included in a block."
+			message, message2 = "", ""
+		case "IMMATURE":
+			title = "This ticket will enter the ticket pool and become a live ticket after 256 blocks (~20 hrs)."
+			message, message2 = "", ""
+		case "LIVE":
+			title = "Waiting to be chosen to vote."
+			message = "The average vote time is 28 days, but can take up to 142 days."
+			message2 = "There is a 0.5% chance of expiring before being chosen to vote (this expiration returns the original ticket price without a reward)."
+		case "VOTED":
+			title = "Congratulations! This ticket has voted."
+			message = "The ticket price + reward will become spendable after 256 blocks (~20 hrs)."
+			message2 = ""
+		case "MISSED":
+			title = "This ticket was chosen to vote, but missed the voting window."
+			message = "Missed tickets will be revoked to return the original ticket price to you."
+			message2 = "If a ticket is not revoked automatically, use the revoke button."
+		case "EXPIRED":
+			title = "This ticket has not been chosen to vote within 40960 blocks, and thus expired. "
+			message = "Expired tickets will be revoked to return the original ticket price to you."
+			message2 = "If a ticket is not revoked automatically, use the revoke button."
+		case "REVOKED":
+			title = "This ticket has been revoked."
+			message = "The ticket price will become spendable after 256 blocks (~20 hrs)."
+			message2 = ""
+		}
+		titleLabel, messageLabel, messageLabel2 := c.theme.Body2(title), c.theme.Body2(message), c.theme.Body2(message2)
+		messageLabel.Color, messageLabel2.Color = c.theme.Color.Gray, c.theme.Color.Gray
+
+		status := c.theme.Body2(t.Info.Status)
+		status.Color = st.color
+		return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+			layout.Rigid(func(gtx C) D {
+				return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						st.icon.Scale = .5
+						return st.icon.Layout(gtx)
+					}),
```

Comment:
You seem to have skipped this. kindly move `st.icon.Scale = .5` to line 675 or to line 641. it really does not need to be where it is.
```suggestion
					layout.Rigid(st.icon.Layout),
```